Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Header.css';
+import ImageThumbnail from './ImageThumbnail.js'
+import HeaderTitle from './HeaderTitle.js'
+import HeaderContent from './HeaderContent.js'
+
+const HeaderContainer = (props) => {
+  return (
+    <div>
+    <div className="headerTitlethumbnail">
```

Comment:
Running a code formatter like [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) over this would fix some of the indentation issues that can pop up when writing JSX, and HTML in general.